src/test/java/ch/puzzle/eft/service/ExamServiceTest.java

```diff
@@ -201,4 +201,20 @@ void shouldReturnCorrectFilesAfterZip() throws IOException {
 
         assertEquals(expectedFileNames.size(), fileCount, "Not all files were zipped correctly.");
     }
+
+    @Test
+    void shouldSortExamList() {
+        List<ExamModel> result = examFileService.getMatchingExams("11001", "22223333");
+
+        List<String> expectedFileNames = List.of("Handels und Gesellschaftsrecht",
+                                                 "Öffentliches Recht",
+                                                 "Privatrecht",
+                                                 "Strafrecht");
+
+        for (int i = 0; i < expectedFileNames.size(); i++) {
```

Comment:
OK, das ist die hard coded Variante.

Es gibt in JUnit und AssertJ Methoden, die den Vergleich von Collections erlauben.